content/tutorials/wallets.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Wallet Basics
+description: 
+---
+
+Ocean users need a wallet to manage their ETH and OCEAN tokens. 
+
+## Recommendations
+
+- **Easiest:** Use MetaMask. See the tutorial about [how to set up MetaMask](/tutorials/metamask-setup/). OR
+- **Still easy, but more secure:** Get a Trezor or Ledger hardware wallet, and use MetaMask to interact with it.
+
+## The Meaning of "Wallet"
```

Comment:
would remove this whole part, it just makes it more confusing. Users need an ERc-20 compatible Ethereum wallet. That's it

Reply:
I think it's quite helpful, because it gives something tangible for newcomers to latch onto. They may not know 'ERC-20' but can still use Metamask. Leaving in for now. If you really really don't want this, re-open this comment.